Графика для всех страниц

rychagoff opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/38/user/675483/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/675483-sedona-38/2/

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/38/user/675483/tasks/7)

rychagoff committed:
Меняет тип отступов для html, css и js файлов

Удобно верстать с табами

rychagoff committed:
Меняет пробелы на табы

rychagoff committed:
Revert "Меняет тип отступов для html, css и js файлов"

This reverts commit 1bcb5e7c21d7ad939ed67dc26b196017ce872766.

rychagoff committed:
Revert "Меняет пробелы на табы"

This reverts commit 3c4453653812e4033e38d40b92e2324a1dafe6fb.

rychagoff committed:
Размечает header, добавляет классы

Разметил шапку, подготовил заготовки для контента и подвала. Прописал классы.

rychagoff committed:
Размечает секцию hero

Разметил первую секцию hero, прописал классы.

rychagoff committed:
Размечает секцию advantages

Разметил секцию преимуществ, прописал классы.

rychagoff committed:
Размечает секцию separator

Разметил первый разделитель, прописал классы.

rychagoff committed:
Размечает секцию services

Разметил секцию услуг, прописал классы. Иконки пока img.

rychagoff committed:
Размечает секцию search

Разметил вторую секцию разделителя с ссылкой на каталог гостиниц.

rychagoff committed:
Размечает секцию subscribe

Разметил секцию подписки, добавил форму, прописал классы.

rychagoff committed:
Размечает footer

Разметил подвал, добавил заготовки под иконки соц.сетей, прописал классы.

rychagoff committed:
Размечает страницу catalog.html

Создал страницу каталога, разметил структуру, прописал классы.

rychagoff committed:
Правит разметку страниц

Поправил ошибки валидации

rychagoff committed:
Правит разметку header

Поправил разметку шапок и прописал классы по БЭМ

rychagoff committed:
Правит разметку footer

Поправил разметку подвалов и прописал классы по БЭМ

rychagoff committed:
Правит разметку main страницы index.html

Поправил разметку главной страницы, добавил обертки для ссылок, прописал классы по БЭМ.

rychagoff committed:
Правит навигацию и секцию filter на странице catalog.html

Поправил разметку ссылок в навигации на страницах, разметку блока с фильтрами на странице каталога.

rychagoff committed:
Правит разметку секции catalog

Поправил разметку карточек товаров и блока с кнопками сортировки и отображения, прописал классы по БЭМ.

rychagoff committed:
Прописывает атрибуты для автопроверки

rychagoff committed:
Правит header после автопроверки

ExTeeP commented on catalog.html:
```diff
@@ -0,0 +1,351 @@
+            <ul class="filter-form__list">
+              <li class="filter-form__item">
+                <label class="filter-form__label">
+                  <input type="radio" name="hotel" checked>Гостиница
```
радио-кнопкам одной группы нужно указать один и тот же name=""
сейчас он разный и можно выбрать все кнопки https://i.imgur.com/NHBBoWG.png

ExTeeP reviewed (approved):
всё хорошо, к след разделу по комменту ток поправить нужно будет

rychagoff committed:
Правит имя радио-кнопок

Поправил радио-кнопки по комментарию из пул-реквеста

rychagoff committed:
Добавляет в проект изображения

Создал папку images, сгруппировал в неё изображения из макета, прописал пути в разметках страниц.
